omim2obo/main.py

```diff
@@ -231,18 +235,55 @@ def run(use_cache: bool = False):
+        cyto_location: str = mim_data['cyto_location']
         if cyto_location:
             # What's 9606chr - joeflack4 2021/11/11
             chr_id = '9606chr' + cyto_location
-            # What's RO['0002525'] - joeflack4 2021/11/11
+            # RO:0002525 (is subsequence of)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
             graph.add((OMIM[mim_number], RO['0002525'], CHR[chr_id]))
+        assocs: Dict = mim_data['phenotype_associations']
+        if len(assocs) == 1:
+            # RO:0003303 (causes condition)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003303
+            # TODO: what to do? This should always be a MIM number, I would think. But there is not always a MIM number
+            assoc = list(assocs.values())[0]
+            pheno_id = assoc['phenotype_identifier']
+            try:
+                pheno_id = int(pheno_id)
+                graph.add((OMIM[mim_number], RO['0003303'], OMIM[pheno_id]))
+            except ValueError:
+                # Report issue (No MIM # In Phenotype label), e.g.
+                #   Phenotype	Gene Symbols	MIM Number	Cyto Location
+                #   3p- syndrome (4)	DEL3pterp25, C3DELpterp25	613792	3pter-p25
+                if 'morbid_map' not in issues:
+                    issues['morbid_map'] = {}
+                if 'issue:nonNumericPhenotypeId' not in issues['morbid_map']:
+                    issues['morbid_map']['issue:nonNumericPhenotypeId'] = {}
+                if pheno_id not in issues['morbid_map']['issue:nonNumericPhenotypeId']:
+                    issues['morbid_map']['issue:nonNumericPhenotypeId'][pheno_id] = {}
+                original_row = f"{assoc['phenotype_label']} ({assoc['phenotype_mapping_info_key']})\t" \
+                               f"{', '.join(mim_data)}\t{mim_number}\t{mim_data['cyto_location']}"
+                issues['morbid_map']['issue:nonNumericPhenotypeId'][pheno_id] = {
+                    'morbidmap.txt_original_row': original_row}
+                # Add triple
+                graph.add((OMIM[mim_number], RO['0003303'], Literal(pheno_id)))
+        # TODO: Determine how to do this properly
+        #   BioLink has a formal way to do this, which we've opted not to do for now. But perhaps in the future
+        #  we can do it. It would involve first the (OMIM[mim_number], BIOLINK['GeneToDiseaseAssociation'],
+        #  BIOLINK['has_evidence']), followed by some way to annotate w/ mim_data['phenotype_mapping_info_label']
+        #  https://biolink.github.io/biolink-model/docs/GeneToDiseaseAssociation.html
+        # graph.add((OMIM[mim_number], BIOLINK['has_evidence'], Literal(mim_data['phenotype_mapping_info_label'])))
+        graph.add((OMIM[mim_number], BIOLINK['GeneToDiseaseAssociation'],
```

Comment:
I saw you're suggestion regarding changing `graph.add((OMIM[mim_number], BIOLINK['has_evidence'], Literal(mim_data['phenotype_mapping_info_label'])))` to something more appropriate. But before I proceed to do that, we should discuss all the issues and nuances with `morbidmap.txt`, as I suspect that your suggestion might change.

I'm temporarily dumping the associations as literal JSON text here so that we can look at the structure as an example.

Reply:
We discussed and have a solution. Marking this resolved.